Templates/ServiceBusTopicTrigger-CSharp/function.json

```diff
@@ -0,0 +1,14 @@
+﻿{
+    "disabled": false,
+    "bindings": [
+        {
+            "name": "mySbMsg",
+            "accessRights" : "Manage",
+            "type": "serviceBusTrigger",
+            "direction": "in",
+            "topicName": "mytopic",
```

Comment:
Then why not make the subscription lower as well? Aren't all the other default values for the various templates lower?